### PR TITLE
Chat flmastor

### PR DESCRIPTION
**Pull request by Mastodoonte**
New push from Justine-chat-branch 
-env_old is deleted
- frame of chat is now responsive

### CODE REVIEW

File: srcs/database/Dockerfile.dev [hunk 1]
```diff
@@ -1,23 +1,24 @@
...
-
-CMD     ["postgres"]
\ No newline at end of file
+COPY ./conf/dev-mock-data.sql ./
+
+COPY ./scripts/docker-entrypoint.dev.sh ./
+
+ENV POSTGRES_USER=transcendence
+ENV POSTGRES_PASSWORD=5euApbMoLVtNgEcKs44e
+ENV POSTGRES_DB=transcendence_db
```
Comment (twagger) [inline srcs/database/Dockerfile.dev:9]: We should not have these variables directly written in the docker file. To be updated.

**Review by twagger — CHANGES_REQUESTED** [review]
Hello, apparently this PR includes a lot of stuff from Chat branch that are not yet finished. You should merge your branch with chat branch instead of PR this on master directly.

To be discussed with Noufel and Justine.

**Review by lefoun — CHANGES_REQUESTED** [review]
We need to merge chat and chat_flmastor together instead of making a pull request. A lot of things need to change in the code it's not production ready.

**Comment by Mastodoonte** [comment]
changes/work will be done right in the appropriate branch in chat

**Pull request closed without merging**